transport/ssh.go

```diff
@@ -212,6 +213,8 @@ type Identity struct {
 	// Retries can be set to a non-zero value to retry every 500ms for that many times.
 	Retries uint `json:"retries"`
 	Port    uint `json:"port"`
+	// If true, skip all host key checking.
+	DisableHostKeyCheck bool `json:"disable_host_key_check"`
```

Comment:
I might be able to do this by making KnownHosts=/dev/null.

Reply:
Whichever way's the one that people are most familiar with works for me.

Reply:
I dislike this approach and hope Bolt moves away from it someday. I think I'm going to remove this option andhack around it with KnownHosts.